packages/loader/driver-utils/src/adapters/compression/summaryblob/documentStorageServiceSummaryBlobCompressionAdapter.ts

```diff
@@ -21,7 +21,10 @@ import {
 import { compress, decompress } from "lz4js";
 
 import { DocumentStorageServiceProxy } from "../../../documentStorageServiceProxy.js";
-import { ICompressionStorageConfig, SummaryCompressionAlgorithm } from "../index.js";
+import {
```

Comment:
Note for reviewers: this is in response to a new rule that prevents importing from parent `index` modules (which can cause import cycles).